### PR TITLE
Add include_num_members support to conversation.info request

### PR DESCRIPTION
**Pull request by Yohan460**
Add [include_num_members](https://api.slack.com/methods/conversations.info#arg_include_num_members) support to [conversation.info](https://api.slack.com/methods/conversations.info) request

### CODE REVIEW

**Comment by Yohan460** [comment]
Hey @kanata2 👋 

Out of curiosity, is there any cadence for releases here? Would really love to get this merged in to resolve a dependency we have.

Many Thanks,

Johan

**Comment by kanata2** [comment]
@Yohan460 
Thanks for submitting PR!
This PR has backward-incompatibility changes.
So we'll release this in the next minor version(= v.10.0) if approved.

File: conversation.go [hunk 1]
```diff
@@ -358,16 +358,17 @@ func (api *Client) CreateConversationContext(ctx context.Context, channelName st
 }
 
 // GetConversationInfo retrieves information about a conversation
-func (api *Client) GetConversationInfo(channelID string, includeLocale bool) (*Channel, error) {
-	return api.GetConversationInfoContext(context.Background(), channelID, includeLocale)
+func (api *Client) GetConversationInfo(channelID string, includeLocale, includeNumMembers bool) (*Channel, error) {
```
Comment (kanata2) [inline conversation.go]: How about making the argument a struct? (like `GetUsersInConversation` method)

**Review by kanata2 — COMMENTED** [review]
@Yohan460 
I've commented again. 
(By the way, the commit seems to be unverified, so please just check)

File: conversation.go (hunk 1 above)
Comment (Yohan460) [reply conversation.go]: Fixed

**Comment by Yohan460** [comment]
Updated the GPG key signing, haven't done pubilc PRs for a long time. The most recent commit should go verified eventually

**Comment by kanata2** [comment]
@Yohan460 It seems that the lint phase in CI failed, so please confirm it.

**Comment by Yohan460** [comment]
@kanata2 The lint failure is un-related to my changes and appears to be due to a failure in the CI/CD pipeline. If you could take a look it would be appreciated

**Comment by kanata2** [comment]
Could you run `gofmt -w conversation.go`?

**Comment by Yohan460** [comment]
done, could you re-trigger CI for me?

**Comment by zchee** [comment]
@Yohan460 Hi, this is newer slack-go maintainer.

I'll review this pull request later. Thanks for contribution.

**Comment by Yohan460** [comment]
bumping this @zchee @kanata2 🥺 🙏

File: conversation.go [hunk 2]
```diff
@@ -357,17 +357,29 @@ func (api *Client) CreateConversationContext(ctx context.Context, channelName st
...
+	return api.GetConversationInfoContext(context.Background(), input)
 }
 
 // GetConversationInfoContext retrieves information about a conversation with a custom context
-func (api *Client) GetConversationInfoContext(ctx context.Context, channelID string, includeLocale bool) (*Channel, error) {
+func (api *Client) GetConversationInfoContext(ctx context.Context, input *GetConversationInfoInput) (*Channel, error) {
+	if input == nil {
+		input = &GetConversationInfoInput{}
+	}
+
```
Comment (kanata2) [inline conversation.go]: The `channelID` must be set, so we should return error here.

**Review by kanata2 — COMMENTED** [review]
Sorry for delay. I plan to release this in v0.11.

File: conversation.go [hunk 3]
```diff
@@ -5,6 +5,8 @@ import (
 	"net/url"
 	"strconv"
 	"strings"
+
+	"github.com/pkg/errors"
```
Comment (kanata2) [inline conversation.go]: We don't use pkg/errors now.
In this case, you can use `fmt.Errorf` or standard errors package instead.

**Review by kanata2 — CHANGES_REQUESTED** [review]

File: conversation.go (hunk 3 above)
Comment (Yohan460) [reply conversation.go]: Changed it to the built in `errors` package

**Review by kanata2 — APPROVED** [review]
Thanks!

File: conversation_test.go [hunk 4]
```diff
@@ -411,6 +413,28 @@ func TestGetConversationInfo(t *testing.T) {
...
+	api := New("testing-token", OptionAPIURL("http://"+serverAddr+"/"))
+	_, err := api.GetConversationInfo(nil)
+	if err == nil {
+		t.Errorf("Unexpected pass where there should have been nil input error")
+		return
+	}
+}
+
+func TestGetConversationInfoMissingChannelError(t *testing.T) {
+	http.HandleFunc("/conversations.info", okChannelJsonHandler)
```
Comment (kanata2) [inline conversation_test.go]: Some tests are failed in CI.
https://github.com/slack-go/slack/runs/6276344051?check_suite_focus=true

**Review by kanata2 — CHANGES_REQUESTED** [review]

File: conversation_test.go (hunk 4 above)
Comment (Yohan460) [reply conversation_test.go]: Argh, this is due to them all sharing the http handler function. I've combined them to resolve this

**Review by kanata2 — APPROVED** [review]
I'm planning to release this in v0.12

**Pull request merged**